### PR TITLE
Added controls for setting gainMax

### PR DESCRIPTION
**Pull request by psmithcrl**
Provides a control and response for auto exposure maximum gain for both the stereo pair, as well as the aux imager.
This provides a limit on the maximum amount of gain the auto exposure algorithm allows.
This is a request based on a customer need to reduce the artifacts within a dark scene.

### CODE REVIEW

File: source/LibMultiSense/include/MultiSense/details/wire/AuxCamConfigMessage.hh [hunk 1]
```diff
@@ -169,6 +172,17 @@ public:
...
 
+        //
+        // version 2 additions
+        if (version >= 2)
+        {
+            message & gainMax;
+        }
+        else
+        {
+            gainMax = std::numeric_limits<float>::max();
```
Comment (mattalvarado) [inline source/LibMultiSense/include/MultiSense/details/wire/AuxCamConfigMessage.hh:183]: Is the max actually a limit. In old systems it was 8

File: source/LibMultiSense/include/MultiSense/MultiSenseTypes.hh [hunk 2]
```diff
@@ -1761,6 +1797,7 @@ public:
                m_profile(User_Control),
                m_gamma(2.0),
                m_sharpeningEnable(false), m_sharpeningPercentage(0.0f), m_sharpeningLimit(0),
+               m_gainMax(std::numeric_limits<float>::max()),
```
Comment (mattalvarado) [inline source/LibMultiSense/include/MultiSense/MultiSenseTypes.hh:1800]: Should we #define the actual max and set that here?

Comment (psmithcrl) [reply source/LibMultiSense/include/MultiSense/MultiSenseTypes.hh]: I just updated this to support the ImagerGainMax which defaults to 1000.0f.

File: source/LibMultiSense/include/MultiSense/details/wire/AuxCamConfigMessage.hh (hunk 1 above)
Comment (psmithcrl) [reply source/LibMultiSense/include/MultiSense/details/wire/AuxCamConfigMessage.hh]: No the greatest max of the imagers is 1000.0f

**Review by mattalvarado — APPROVED** [review]

**Pull request merged**